.circleci/config.yml

```diff
@@ -100,7 +100,7 @@ jobs:
             npm ci
             npm test
             mkdir -p coverage-alpine
-            cp coverage/* coverage-alpine/
+            cp -r coverage/* coverage-alpine/
```

Comment:
```suggestion
            cp -r coverage/ coverage-alpine/
```

Reply:
That actually broke Alpine https://app.circleci.com/pipelines/github/codecov/uploader/738/workflows/09e0e139-b0c4-4492-b702-e90d4af53928/jobs/5134

Seems the copy didn;t find the coverage report with the change https://app.circleci.com/pipelines/github/codecov/uploader/738/workflows/09e0e139-b0c4-4492-b702-e90d4af53928/jobs/5130

Reply:
I am very curious as to why that might be. This is a good candidate for `git revert`.